src/pages/docs/api/realtime-sdk.mdx

```diff
@@ -0,0 +1,1637 @@
+
+`Ably::Realtime.new(ClientOptions client_options)`
+
+</If>
+
+<If lang="java">
+
+`new io.ably.lib.AblyRealtime(ClientOptions clientOptions)`
+
+</If>
+
+<If lang="csharp">
+
+`new AblyRealtime(ClientOptions clientOptions)`
+
+</If>
+
+<If lang="objc">
+
+`(instancetype)initWithOptions:(ARTClientOptions *)options;`
+
+</If>
+
+<If lang="swift">
+
+`init(options: ARTClientOptions)`
+
+</If>
+
+<If lang="flutter">
+
+`ably.Realtime(options: ClientOptions clientOptions)`
+
+</If>
+
+<If lang="javascript,nodejs,java,swift,objc,csharp,flutter">
+
+This will instantiate the library using the specified [ClientOptions](#client-options).
+
+</If>
+
+<If lang="ruby">
+
+This will instantiate the library and create a new `Ably::Realtime::Client` using the specified [ClientOptions](#client-options).
+
+</If>
+
+<If lang="javascript,nodejs">
+
+`new Ably.Realtime(String keyOrTokenId)`
+
+</If>
+
+<If lang="ruby">
+
+`Ably::Realtime.new(String key_or_token_id)`
+
+</If>
+
+<If lang="java">
+
+`new io.ably.lib.AblyRealtime(String keyOrTokenIdString)`
+
+</If>
+
+<If lang="csharp">
+
+`new AblyRealtime(string key)`
+
+</If>
+
+<If lang="objc">
+
+`(instancetype)initWithKey:(NSString *)key
+(instancetype)initWithToken:(NSString *)token`
+
+</If>
+
+<If lang="swift">
+
+`init(key: String)
+init(token: String)`
+
+</If>
+
+<If lang="flutter">
+
+`ably.Realtime(key: String key)`
+
+</If>
+
+This will instantiate the Realtime library with the provided API key or Token ID string.
+
+The Realtime constructor is used to instantiate the library. The Realtime library may be instantiated multiple times with the same or different [`ClientOptions`](#client-options) in any given context. Except where specified otherwise, instances operate independently of one another.
+
+### Authentication <a id="authentication" />
+
+The Realtime library needs to have credentials to be able to authenticate with the Ably service. Ably supports both Basic and Token based authentication schemes. Read more on [authentication](/docs/auth).
+
+#### Basic Authentication
+
+<If lang="javascript,nodejs,ruby,java,swift,objc,flutter">
+
+You can pass a full-length API key in as `ClientOptions#key` (or just straight into the constructor instead of a `ClientOptions` instance), as obtained from the [application dashboard](https://ably.com/dashboard). Use this option if you wish to use [Basic authentication](/docs/auth/basic), or if you want to be able to [request Ably Tokens](/docs/auth/token) without needing to defer to a separate entity to sign [Ably TokenRequests](/docs/api/realtime-sdk/authentication#token-request). Note that initializing the library with a `key` does not necessarily mean that the library will use Basic auth; it is also able to create and sign Ably [TokenRequests](/docs/api/realtime-sdk/authentication#token-request), and can use token authentication for itself if it needs to or if [`ClientOptions#useTokenAuth`](#client-options) is enabled.
+
+</If>
+
+<If lang="csharp">
```

Comment:
Could this be added to the above <If lang="javascript,nodejs,java,swift,objc,flutter"> ??


Note: it renders fine locally